R/score_model_out.R

```diff
@@ -72,7 +78,11 @@
 #' )
 #' head(pmf_scores)
 #'
-#' @return forecast_quantile
+#' @return A data.table with scores
```

Comment:
Technically it's an object of class `scores` which is itself a `data.table` but probably easiest not to confuse users with `scoringutils` stuff